crates/static-analysis-kernel/src/analysis/ddsa_lib/bridge/ts_node.rs

```diff
@@ -153,57 +153,17 @@ mod tests {
     use crate::analysis::ddsa_lib::bridge::ts_node::TsNodeBridge;
     use crate::analysis::ddsa_lib::bridge::ContextBridge;
     use crate::analysis::ddsa_lib::common::get_field;
-    use crate::analysis::ddsa_lib::test_utils::{attach_as_global, cfg_test_runtime, try_execute};
+    use crate::analysis::ddsa_lib::test_utils::{
+        attach_as_global, cfg_test_runtime, try_execute, TsTree,
+    };
     use crate::analysis::ddsa_lib::RawTSNode;
-    use crate::analysis::tree_sitter::get_tree_sitter_language;
     use crate::model::common::Language;
     use deno_core::v8;
     use deno_core::v8::HandleScope;
     use std::cell::RefCell;
     use std::rc::Rc;
     use std::sync::Arc;
 
-    /// A tree-sitter tree and the text it was parsed from, along with convenience functions
-    /// to inspect the nodes.
-    struct Tree(Arc<tree_sitter::Tree>, &'static str);
-
-    impl Tree {
-        fn text(&self, node: tree_sitter::Node<'_>) -> &'static str {
-            self.1.get(node.start_byte()..node.end_byte()).unwrap()
-        }
-
-        /// Shorthand to extract the first TSNode with matching text and TSSymbol.
-        fn find_first<'t>(&'t self, text: &str, kind: &str) -> tree_sitter::Node<'t> {
-            let mut cursor = self.0.walk();
-            let root = cursor.node().id();
-            loop {
-                cursor.goto_descendant(cursor.descendant_index() + 1);
-                let node = cursor.node();
-                if node.id() == root {
-                    break;
-                }
-                if (kind == "_" || node.grammar_name() == kind) && self.text(node) == text {
-                    return node;
-                }
-            }
-            panic!("no `({})` nodes matched text \"{}\"", kind, text);
-        }
-    }
-
-    /// A newtype [`tree_sitter::Parser`] that constructs a [`Tree`].
-    struct TsParser(tree_sitter::Parser);
-    impl TsParser {
-        fn new(language: Language) -> Self {
-            let lang = get_tree_sitter_language(&language);
-            let mut parser = tree_sitter::Parser::new();
-            parser.set_language(&lang).unwrap();
-            Self(parser)
-        }
-        fn parse(&mut self, text: &'static str) -> Tree {
-            Tree(Arc::new(self.0.parse(text, None).unwrap()), text)
-        }
-    }
-
```

Comment:
What is the reason for removing this from the core and putting only in the tests?